akka-http/src/main/scala/akka/http/scaladsl/server/directives/CodingDirectives.scala

```diff
@@ -134,7 +134,7 @@ trait CodingDirectives {
 
   /**
    * Inspects the response entity and adds a `Content-Encoding: gzip` response header if
-   * the entities media-type is precompressed with gzip and no `Content-Encoding` header is present yet.
+   * the entity's media-type is precompressed with gzip and no `Content-Encoding` header is present yet.
```

Comment:
entity's = possessive singular
entities' = possessive plural 

But confusingly:
it's = it is
its = possessive